plugins/UsersManager/API.php

```diff
@@ -879,6 +879,11 @@ private function enrichUser($user)
         unset($user['password']);
         unset($user['ts_password_modified']);
 
+        if($lastSeen = LastSeenTimeLogger::getLastSeenTimeForUser($user['login']))
+        {
+            $user['date_last_seen'] =  gmdate("Y-m-d\TH:i:s\Z", $lastSeen);
```

Comment:
I guess we use to return the date format `Y-m-d H:i:s` in other places. See https://github.com/matomo-org/matomo/blob/c973567705a0065fdd7d7c7b11b80f1f0f1be350/core/Date.php#L389-L398